ZUORA_VERSION

```diff
@@ -0,0 +1 @@
+0.0.2-sage
```

Comment:
Is this file needed?

Reply:
I found it easier to follow the pattern used for retrieving the version in the eventq gem. Was having a bit of trouble with how the Zuora Gem version is been defined and retrieved when setting up the container and it was getting messy to get it to work when setting up the docker container (edited).

Was going to delete this [file](https://github.com/Sage/zuora/blob/master/lib/zuora/version.rb) after going with the new pattern, but missed it.

Thoughts?

Reply:
Fixed in this commit [497b6d4](https://github.com/Sage/zuora/pull/14/commits/497b6d4dd7316909011bb1f950e2d4812b7046bf)